collections/high_availability/roles/drbd/tasks/main.yml

```diff
@@ -1,4 +1,4 @@
-- name: "package <|> Install packages"
+- name: package █ Install DRBD packages
```

Comment:
Can you replace the █ by <|> in the whole file ?
Reason is: these ASCII extended characters generates issues on some shells, so removed them. 🙂

Reply:
Done.